### DISCUSSION TITLE
How to use the anti-satellite missile?

### DISCUSSION THREAD

**Discussion by vincemolnar** [carriontrooper/Alpha-Frontier#54] [Q&A] [answered]
I'm being bullied by all my neighbors all through the game. Most of them have invented nuclear weapons by now, and I've got the first nuclear satellite attack which basically wiped me out as I'm going for a single city challenge. I thought anti-satellite missiles will be able to strike these somehow, but I can't find a way to launch a preventive attack or intercept their attack. Moreover, this bastard can strike multiple times so I have zero chance after the first strike to rebuild.

So, how do these anti-satellite things work?

**Comment by carriontrooper** [comment, marked as answer]
Yeah, just fixed a bug which left the anti-sat missiles unable to intercept - they should be able to intercept Satellite units now, hopefully. Do test and tell me if they still don't work. Alternatively, Defense Satellite can also intercept nukes (I think), but they're a bit later on the tech tree.

**Comment by Dardjan** [comment]
А когда будет новое обновление?

**Reply by carriontrooper** [reply]
Maybe soon, graphics-wise. I'm currently working on AbsoluteUnits version of Alpha Frontier sprites - once that's done, I plan on making HexaRealm version of the tiles.
And maybe an idea from discord about a new natural wonder too perhaps.
